invokeai/backend/stable_diffusion/diffusion/custom_atttention.py

```diff
@@ -188,11 +208,12 @@ def __call__(
         hidden_states = attn.to_out[1](hidden_states)
 
         if input_ndim == 4:
+            batch_size, channel, height, width = hidden_states.shape
             hidden_states = hidden_states.transpose(-1, -2).reshape(batch_size, channel, height, width)
 
         if attn.residual_connection:
             hidden_states = hidden_states + residual
 
         hidden_states = hidden_states / attn.rescale_output_factor
 
-        return hidden_states
+        return cast(torch.FloatTensor, hidden_states)
```

Comment:
This block has a comment: `Start unmodified block from AttnProcessor2_0.`

Probably should remove that bc it is now modified

Reply:
Will clean up once Ryan approves the PR.